gpu-operator/life-cycle-policy.rst

```diff
@@ -89,26 +89,26 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
      - ${version}
 
    * - NVIDIA GPU Driver
-     - | `570.133.20 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-133-20/index.html>`_ (recommended), 
-       | `570.124.06 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-124-06/index.html>`_ (default)
-       | `570.86.15 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-86-15/index.html>`_ 
+     - | `570.133.20 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-133-20/index.html>`_ (default, recommended)
```

Comment:
will there be another driver version here?

Reply:
We will likely be adding 570.148.08 once we release driver containers for it.

Reply:
@cdesiniotis or @tariq1890 will 570.148.08 be the recommended/default version for this release?

Reply:
`570.148.08` will the default and recommended version for the v25.3.1 release